.pre-commit-config.yaml

```diff
@@ -34,18 +34,15 @@ repos:
     rev: v0.942
     hooks:
       - id: mypy
-        args:
-          - --ignore-missing-imports
-          # Silence errors about Python 3.9-style delayed type annotations on Python 3.8
-          - --python-version
-          - "3.9"
+        args: []
```

Comment:
Override default ``--ignore-missing-imports``

Reply:
Maybe worthwhile mentioning this as a comment in the YAML

Reply:
done